src/configs/subexports/index.js

```diff
@@ -0,0 +1 @@
+export default {}
```

Comment:
This is the file that should have the loadConfig part. Check the other config files.